src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php

```diff
@@ -36,10 +36,9 @@
     protected $drivers = array();
 
     /**
-     * @param array            $objectManager A configured object manager
-     * @param ContainerBuilder $container     A ContainerBuilder instance
+     * @param array $objectManager A configured object manager
      *
-     * @throws \InvalidArgumentException
+     * @throws InvalidArgumentException
```

Comment:
@TomasVotruba Not sure why it removed the blackshash here.

Reply:
Fixed: https://github.com/Symplify/Symplify/pull/602

Please try [v3.2.5](https://github.com/Symplify/BetterReflectionDocBlock/releases/tag/v3.2.5)

Reply:
@TomasVotruba Running the new version give me this error:

```
  Fixing of "src/Symfony/Component/HttpFoundation/Request.php" file by "Symplify\CodingStandard\Fixer\Commenting\RemoveUselessDocBlockFixer" failed: Unknown modifier 'c' in pattern: #@see[\s]+(?<has_
  slash>\\)?https\://tools\.ietf\.org/html/rfc7231#section\-4\.2\.1# in file /Users/fabien/Code/github/Symplify/EasyCodingStandard/vendor/nette/utils/src/Utils/Strings.php on line 604
```

Reply:
Fixed in 3.2.6

Here is preview https://github.com/symfony/symfony/compare/2.7...TomasVotruba:phpdocs-removal

of:

```yml
# ecs.neon

checkers:
    - PhpCsFixer\Fixer\Phpdoc\PhpdocAlignFixer
    - Symplify\CodingStandard\Fixer\Commenting\RemoveUselessDocBlockFixer

    # remove extra blank lines caused by previous fixer
    - Symplify\CodingStandard\Fixer\Commenting\RemoveEmptyDocBlockFixer
    - Symplify\CodingStandard\Fixer\Commenting\RemoveSuperfluousDocBlockWhitespaceFixer
```

Reply:
I see there are few edge case. I'll try to fix some, but some are very difficult to catch with provided ReflectionDocBlock api.